### PR TITLE
Dead Pull - Microsoft.PowershellPreview version 7.2.0.10

### PR DESCRIPTION
**Pull request by Masamune3210**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28942)

### CODE REVIEW

File: manifests/m/Microsoft/PowershellPreview/7.2.0.10/Microsoft.PowershellPreview.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PowershellPreview
```
Comment (jedieaston) [inline manifests/m/Microsoft/PowershellPreview/7.2.0.10/Microsoft.PowershellPreview.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.PowerShell.Preview
```

File: manifests/m/Microsoft/PowershellPreview/7.2.0.10/Microsoft.PowershellPreview.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PowershellPreview
```
Comment (jedieaston) [inline manifests/m/Microsoft/PowershellPreview/7.2.0.10/Microsoft.PowershellPreview.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.PowerShell.Preview
```

File: manifests/m/Microsoft/PowershellPreview/7.2.0.10/Microsoft.PowershellPreview.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PowershellPreview
```
Comment (jedieaston) [inline manifests/m/Microsoft/PowershellPreview/7.2.0.10/Microsoft.PowershellPreview.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.Powershell.Preview
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
And, of course, move it to the correct folder.

(Or just run `wingetcreate update` again with the ID `Microsoft.PowerShell.Preview`).

**Comment by Masamune3210** [comment]
Whats weird is I opened this using wingetcreate update microsoft.powershell.preview -i so I figured that everything would be up to snuff already. Any idea why It got the id wrong?

**Comment by jedieaston** [comment]
I don't know why. That's really weird.

**Comment by Masamune3210** [comment]
Tried it again, and it made a preview folder inside of the powershell folder correctly this time, hopefully thirds the charm lol

**Pull request closed without merging**